_topic_map.yml

```diff
@@ -754,21 +754,23 @@ Topics:
   File: setting-up-trusted-ca
   Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
 ---
-#Name: OpenShift Pipelines
-#Dir: openshift_pipelines
-#Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#Topics:
+Name: Pipelines
+Dir: openshift_pipelines
+Distros: openshift-enterprise,openshift-webscale,openshift-origin
+Topics:
 #- Name: Understanding OpenShift Pipelines
 #  File: understanding-openshift-pipelines
 #- Name: Installing Pipelines
 #  File: installing-pipelines
-#- Name: Uninstalling Pipelines
-#  File: uninstalling-pipelines
+- Name: Uninstalling Pipelines
+  File: uninstalling-pipelines
 #- Name: Getting started with OpenShift Pipelines
 #  File: getting-started-with-openshift-pipelines
 #- Name: Working with Pipelines using the Developer perspective
 #  File: working-with-pipelines-using-the-developer-perspective
-#---
+- Name: Release Notes
```

Comment:
Should be "Release notes" to be sentence case. I might also suggest to be more specific, like "OpenShift Pipelines release notes"

Reply:
Hi @bergerhoffer makes sense, we will make these changes through this PR for the Topic map changes: https://github.com/openshift/openshift-docs/pull/21932 
This will be merged on the day of the TP release.